lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -434,6 +433,7 @@ import qualified Cardano.Wallet.Primitive.Types.TokenMap as TokenMap
 import qualified Cardano.Wallet.Primitive.Types.Tx.Gen as TxGen
 import qualified Cardano.Wallet.Primitive.Types.UTxO as UTxO
 import qualified Cardano.Wallet.Primitive.Types.UTxOIndex as UTxOIndex
+import Cardano.Wallet.Read.Primitive.Tx.Features.Integrity
```

Comment:
```suggestion

import Cardano.Wallet.Read.Primitive.Tx.Features.Integrity
    ( txIntegrityCardanoApi )

```

Style: Import list: Either qualified, or explicit import that goes into the other group.

Reply:
sometimes they slip through, formatting automation should help here ...